ReadMe.md

```diff
@@ -2,13 +2,21 @@
 This repository is a contained repository to run Pytorch Lightning (PTL from here on) sample on AWS Trainium. It was developed by taking only the relevant code from <a href=https://awsdocs-neuron.readthedocs-hosted.com/en/latest/libraries/neuronx-distributed/tutorials/training_llama2_tp_pp_ptl.html>this</a> tutorial. The original tutorial uses parallel cluster to run the workload on mulitple node cluster. This is a simpler sample that can train LLama 2.0 7B model on a single trn1.32xlarge instance with reduced batch size and using all cores for Tensor Parallelism.
 It also contains a hacky implementation to make Ray work with PTL on Trainium.
 
-## Prerequisites / Setup (Needed commonly for all subsequent sections)
+## Common Prerequisites / Setup (Needed commonly for all subsequent sections)
 - Get a single trn1.32xlarge AWS Trainium EC2 instance, use the Neuron DLAMI when creating the instance (refer to ```EC2-Instance-Screenshot.png``` if you are new to this).
 - Clone this repository (The commands below assume the repository is cloned in home directory at path /home/ubuntu. If a different path is used to clone, the commands need to be changed accordingly).
+
+
+## LLama 2.0 Pre-requisites:
 - Download the dataset. The command below downloads the dataset to the path ```/home/ubuntu/examples_datasets/``` and further commands in this ReadMe assume that path for dataset. If the script is modified OR the data is placed in some other path, the commands need to be adjusted as appropriate.
 ```
 cd neuron-lightning-ray/data && python get_dataset.py --llama-version 2
 ```
+- Install the requirements
+```
+pip install -r requirements.txt
+Install neuronx-distributed: python -m pip install neuronx_distributed --extra-index-url https://pip.repos.neuron.amazonaws.com
```

Comment:
This shouldn't be needed when we use the DLAMI right? If not, there are more installment steps required.

Reply:
i dont think with DLAMI neuronx-distributed is included and the requirements.txt has ray and correct transformer version in it, which are essential.

Reply:
I don't remember installing `neuronx-distributed` but I could have forgotten. Let me approve the PR so that you can merge.